### DISCUSSION TITLE
Library Suggestion for Push Notifications

### DISCUSSION THREAD

**Discussion by no-1ne** [lexiconhq/lexicon#34] [Ideas]
Thank you for sharing your work with everyone else the apps looks amazing, 

coming to the issue, docs show push notifications status  work in progress. Can this lib help in implementing them https://github.com/invertase/notifee

**Comment by JuliaAgostino** [comment]
Is this project still being actively worked on?

**Comment by no-1ne** [comment]
Let's wait and see, seems like it's little slow

**Comment by JuliaAgostino** [comment]
> Let's wait and see, seems like it's little slow

Yeah, very slow... which is quite concerning because I really like this project so far.

**Comment by no-1ne** [comment]
It's very tricky maintaining open source project while keeping the lights on in the office. community contributions can be a motivating factor if the project owners can share the roadmap, community can jump on things which they aren't working on or can use some assistance

**Comment by JuliaAgostino** [comment]
> It's very tricky maintaining open source project while keeping the lights on in the office. community contributions can be a motivating factor if the project owners can share the roadmap, community can jump on things which they aren't working on or can use some assistance

Yes, exactly! A proper roadmap would be really nice.

**Comment by Kinostrome** [comment]
Thanks for the suggestion @no-1ne, it's very appreciated. Our implementation of push notifications is specific to Expo. If we find use for this library as part of that, we'll be sure to check it out.
